clang/include/clang/Basic/Builtins.def

```diff
@@ -494,6 +494,9 @@ BUILTIN(__builtin_isinf,      "i.", "FnctE")
 BUILTIN(__builtin_isinf_sign, "i.", "FnctE")
 BUILTIN(__builtin_isnan,      "i.", "FnctE")
 BUILTIN(__builtin_isnormal,   "i.", "FnctE")
+BUILTIN(__builtin_issubnormal,"i.", "FnctE")
+BUILTIN(__builtin_iszero,     "i.", "FnctE")
+BUILTIN(__builtin_issignaling,"i.", "FnctE")
```

Comment:
The `E` means they can be evaluated at compile time, but I don't see changes to `ExprConstant.cpp`?

Reply:
Yes, this is a copy-past error. Thanks for the catch.